.prettierignore

```diff
@@ -9,4 +9,4 @@ app/vendor/**
 .nyc_output/**
 .vscode/**
 test/e2e/send-eth-with-private-key-test/**
-*.scss
+# *.scss
```

Comment:
Sorry I missed what the decision was to do on this? I thought we were just going to remove from the PR?

Reply:
Yes, going to remove -- thanks for the reminder!